lm_test_monitoring.md

```diff
@@ -0,0 +1,231 @@
+
+* monitor the status of the automated tests you have configured to run on your
+  workflow;
+* trigger the execution of those tests if too much time has passed since their
+  previous execution.
+
+![LM monitored workflows](./images/lm_web_workflows.png)
+
+Thus, using LM implies that you have created tests for your workflow and that
+you have created an automated pipeline to execute them using a [compatible
+testing service](#compatible-testing-services).  If you're not this far yet with
+your workflow, you can have a look at our [generating workflow testing
+tips](./reference_general_workflow_testing_tips).
+
+## Compatible Testing Services
+
+LifeMonitor currently supports monitoring tests that run on [GitHub
+Actions](https://docs.github.com/en/actions) and
+[Jenkins](https://www.jenkins.io/).
+
+## Registering your workflow through the LM GitHub app
+
+The [LM GitHub
+app](./lm_wft_best_practices_github_app#the-lifemonitor-github-app) can
+automatically register your workflow testing pipeline with the LifeMonitor
+service.
+
+If you use a repository structure that follows best practices it should be able
+to automatically extract all the necessary metadata. If you want to use a custom
+repository structure then you'll need to provide the app with the metadata
+required to create a [Workflow Testing RO-Crate](./workflow_testing_ro_crate).
+It may be possible to do this by answering the bot's questions through GitHub
+issues; alternatively, you can [manually register your
+workflow](#manually-registering-your-workflow).
+
+## Manually registering your workflow
+
+If required, you can manually register your workflow's test pipeline(s) with the
+LM service (e.g., if your workflow repository is not on GitHub, or you need to
+support a custom repository layout).
+
+### Key Concepts
+
+Manually registering your workflow testing pipeline requires creating a
+[Workflow Testing
+RO-Crate](https://www.lifemonitor.eu/workflow_testing_ro_crate) (WTROC) file that
+captures the required metadata (e.g., where do the tests run?). The WTROC is a
+specialization of a [Workflow
+RO-Crate](https://about.workflowhub.eu/Workflow-RO-Crate/).  You can find an
+in-depth description on its [dedicated
+page](https://www.lifemonitor.eu/workflow_testing_ro_crate).
+
+Not all entities defined by the WTROC profile are required, though the more
+metadata you include the better you will be able to use LM's functionality
+(i.e., some things can't work if you don't provide the required metadata). The
+LifeMonitor requires the definition of at least the following entities.
+
+* Main Workflow: the workflow which is being tested.
+* TestSuite: a set of tests for the workflow.
+* TestService: a software service on which the testing pipeline runs (e.g.,
+  GitHub Actions, your institutional Jenkins installation, etc.).
+* TestInstance: a specific testing pipeline which runs a TestSuite on a specific
+  TestService -- i.e., an instantiation of a TestSuite.
+
+```mermaid
```

Comment:
This is not rendered

Reply:
fixed this in commit [0cd780e](https://github.com/crs4/life_monitor/pull/257/commits/0cd780eba12ea15d2a5aab921f093d1bc4224ffd)